src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -234,8 +231,8 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Include="--target=$(TargetTriple)" Condition="'$(TargetTriple)' != ''" />
       <LinkerArg Include="-g" Condition="$(NativeDebugSymbols) == 'true'" />
       <LinkerArg Include="-Wl,--strip-debug" Condition="$(NativeDebugSymbols) != 'true' and '$(_IsApplePlatform)' != 'true'" />
-      <LinkerArg Include="-Wl,-rpath,'$(IlcRPath)'" Condition="'$(StaticExecutable)' != 'true' and !$([MSBuild]::IsOSPlatform('Windows'))" />
-      <LinkerArg Include="-Wl,-rpath,&quot;$(IlcRPath)&quot;" Condition="'$(StaticExecutable)' != 'true' and $([MSBuild]::IsOSPlatform('Windows'))" />
+      <LinkerArg Include="-Wl,-rpath,'$(IlcRPath)'" Condition="'$(IlcRPath)' != '' and '$(StaticExecutable)' != 'true' and !$([MSBuild]::IsOSPlatform('Windows'))" />
+      <LinkerArg Include="-Wl,-rpath,&quot;$(IlcRPath)&quot;" Condition="'$(IlcRPath)' != '' and '$(StaticExecutable)' != 'true' and $([MSBuild]::IsOSPlatform('Windows'))" />
```

Comment:
I think we can just deleted these as it is unlikely for people to know the symbolic values per platform than to just provide `<LinkerArg Include="-Wl,-rpath...` in their own projects as they do for all linker options which we don't cover.